docker/bookkeeper/Dockerfile

```diff
@@ -36,7 +36,7 @@ RUN set -x \
     && cp -r bookkeeper-all-${BK_VERSION}/* /opt/bookkeeper/ \
     && rm -rf "bookkeeper-all-${BK_VERSION}" "$DISTRO_NAME.tar.gz" "$DISTRO_NAME.tar.gz.asc" "$DISTRO_NAME.tar.gz.sha512" \
     # install zookeeper shell
-    && wget -q https://bootstrap.pypa.io/get-pip.py \
+    && wget -q https://bootstrap.pypa.io/2.7/get-pip.py \
```

Comment:
One question: This image is based on `apache/bookkeeper:4.11.1`, which already has installed `zk-shell`, right? So my question is, can we get rid of install pip at all? `zk-shell` should be already available, if I'm not wrong.

Reply:
Great catch! You're right - base image `apache/bookkeeper:4.11.1` has zk-shell installed out of the box, thus I think it'd be safe to remove zk-shell from the Dockerfile.
I haven't found any `pip` usage apart from `zk-shell` installation, thus I think `pip` references could be removed from Dockerfile as well.
```
$ docker run -it --entrypoint /bin/bash apache/bookkeeper:4.11.1
...
4.11.1: Pulling from docker.io/apache/bookkeeper
...
Status: Downloaded newer image for docker.io/apache/bookkeeper:4.11.1
[root@e544aa0b1b8c bookkeeper]# zk-shell
Welcome to zk-shell (1.3.2)
(DISCONNECTED) />
```